src/Microsoft.ComponentDetection.Detectors/linux/Contracts/SyftOutput.cs

```diff
@@ -371,6 +371,22 @@ public partial struct License
 
     public static implicit operator License(string String) => new License { String = String };
     public static implicit operator License(string[] StringArray) => new License { StringArray = StringArray };
+
+    // Override ToString method to provide a string representation of the struct
+    public override string ToString()
```

Comment:
This whole file is autogenerated. I would add a `SyftExtensions` file or class, and add an extension method for `License` instead.

Reply:
Turns out this was actually unnecessary. I Removed the changes in this file.